.travis.yml

```diff
@@ -7,6 +7,7 @@ notifications:
   email: false
 node_js:
   - 'node'
+  - '7'
```

Comment:
As ESLint still supports Node 4, I'm with you on that.  
Do *you* have any idea about Travis failing when using v8?